packages/py-api/py_api/models/hackathon_participants_models.py

```diff
@@ -1,8 +1,25 @@
-from typing import Literal, Optional
+from typing import Annotated, Literal, Optional
 
-from pydantic import BaseModel, EmailStr
+from pydantic import BaseModel, EmailStr, Field, validator
 
-TSHIRT_SIZE = Literal["S", "M", "L"]
+TSHIRT_SIZE = Literal["S", "M", "L", "XL"]
+UNIVERSITIES_LIST = Literal[
+    "Sofia University",
+    "Technical University - Sofia", "Plovdiv University", "Other",
+]
+ALLOWED_AGE = Annotated[int, Field(ge=0, le=99)]
+REFERRAL_SOURCES_LIST = Literal[
+    "University",
+    "Friends", "I was on a previous edition of Hack AUBG",
```

Comment:
Please put new lines between the constants as they are a bit long. This is for every constant.